openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIRequestBodyMapper.java

```diff
@@ -228,19 +243,25 @@ private void handleMultipleMIMETypes(RequestBody bodyParameter,
                                          RequiredParameterNode payloadNode, Map<String, Schema> schema) {
 
         for (MappingFieldNode fieldNode: fields) {
-            if (fieldNode.children() != null) {
-                ChildNodeList nodeList = fieldNode.children();
-                for (Node nextNode : nodeList) {
-                    if (nextNode instanceof ListConstructorExpressionNode) {
-                        SeparatedNodeList mimeList = ((ListConstructorExpressionNode) nextNode).expressions();
-                        if (mimeList.size() != 0) {
-                            RequestBody requestBody = new RequestBody();
-                            for (Object mime : mimeList) {
-                                if (mime instanceof BasicLiteralNode) {
-                                    createRequestBody(bodyParameter, payloadNode, schema, requestBody,
-                                            (BasicLiteralNode) mime);
+            if (((SpecificFieldNode) fieldNode).fieldName().toString().equals(MEDIA_TYPE)) {
+                if (fieldNode.children() != null) {
```

Comment:
Shall we invert this if condition and use `continue` clause to reduce the indentation levels?